crates/sqlexec/src/engine.rs

```diff
@@ -321,6 +322,7 @@ pub struct Engine {
     task_scheduler: Scheduler,
     /// Task executors.
     _task_executors: Vec<TaskExecutor>,
+    _tmp_dir: Option<tempfile::TempDir>,
```

Comment:
Remove underscore (also should probably remove underscore for `_task_executors` too).

Reply:
done, `_task_executors` still isn't used